xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java

```diff
@@ -122,6 +122,19 @@ public static Order adaptOrder(GeminiOrderStatusResponse geminiOrderStatusRespon
     OrderStatus orderStatus = adaptOrderstatus(geminiOrderStatusResponse);
     Date timestamp = new Date(geminiOrderStatusResponse.getTimestampms());
 
+    // Calculate Fees in counter currency
+    BigDecimal fee = null;
```

Comment:
set this to BigDecimal.ZERO and then in your loop you can just add for all and it becomes a 1-liner

Reply:
I ran into issues with the scale of the BigDecimal when setting to BigDecimal.ZERO originally.
Example, adding 0 to 0.00023 would still result in a BigDecimal displaying as 0. Tried to address this by setting the scale before and after the calculations, but it didn't seem to work.

Reply:
I was debugging the BigDecimal returned using IntelliJ's debugger.